### PR TITLE
Prepare for rust stable 1.60

### PR DESCRIPTION
**Pull request by bkchr**
Besides preparing the UI tests this also adds a new script update-rust-stable.sh script for
simplifying the update of a rust stable version. This script will run all UI tests for the new
rust stable version and updating the expected output.

### CODE REVIEW

File: primitives/api/test/tests/trybuild.rs [hunk 1]
```diff
@@ -20,6 +20,11 @@ use std::env;
 #[rustversion::attr(not(stable), ignore)]
```
Comment (ggwpez) [inline primitives/api/test/tests/trybuild.rs:20]: These are probably not needed any longer, as `RUN_UI_TESTS` can be used to toggle the execution.

**Review by ggwpez — APPROVED** [review]
So hopefully I won't have to copy&paste the CI error output in the future anymore :laughing:

File: primitives/api/test/tests/trybuild.rs (hunk 1 above)
Comment (bkchr) [reply primitives/api/test/tests/trybuild.rs:20]: Yeah, but then someone could still run it with a nightly. I would leave it.

**Comment by bkchr** [comment]
@alvicsam I finally created the corresponding polkadot pr: https://github.com/paritytech/polkadot/pull/5282

Sorry for the delay! In the meanwhile rust has released 1.60. Could you please update the image directly to 1.60. Ty!

**Comment by alvicsam** [comment]
@bkchr `staging` image has already 1.60 version. However, the nightly version is updated manually. Currently it's 1.58 (`nightly-2021-11-08`), should I update it to 1.62 (`nightly-2022-04-08`)?

**Comment by bkchr** [comment]
Yeah, please also update the nightly.

**Comment by alvicsam** [comment]
Nightly rust is updated, the latest pipeline used stable 1.60 and nightly 1.62

**Review by alvicsam — APPROVED** [review]

**Pull request merged**